store.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+type ToDoStore struct {
+	Tasks []Task
+	Count int // total number of tasks (used as ID for individual task)
+}
+
+func NewToDoStore() *ToDoStore {
+	return &ToDoStore{}
+}
+
+func (store *ToDoStore) Add(task *Task) {
+	store.Count += 1
+	task.ID = store.Count
```

Comment:
Something to think about is how you would support deletions. If you use the count as an ID, you may run into issues depending on how you implement deletions.

Reply:
Maybe `Count` is misleading. It only ever increments so tasks should have unique ID